django_prices/models.py

```diff
@@ -10,67 +11,105 @@
 from .validators import MoneyPrecisionValidator
 
 
-class MoneyField(models.DecimalField):
+class MoneyField:
 
-    description = "A field that stores an amount of money"
+    description = (
+        "A field that combines an amount of money and currency code into Money"
+        "It allows to store prices with different currencies in one database."
+    )
+    empty_values = list(validators.EMPTY_VALUES)
 
-    def __init__(self, verbose_name=None, currency=None, **kwargs):
-        self.currency = currency
-        super(MoneyField, self).__init__(verbose_name, **kwargs)
+    # Field flags
+    auto_created = False
+    blank = True
+    concrete = False
+    editable = False
 
-    def from_db_value(self, value, expression, connection, context):
-        return self.to_python(value)
+    is_relation = False
+    remote_field = None
```

Comment:
Shouldn't these become a base class that MoneyField and TaxedMoneyField extend? Something like NonDatabaseFieldBase?